src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+import { useNavigate, Link } from 'react-router-dom';
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+
+const Home = (props) => {
+  let navigate = useNavigate();
+  const { token, setToken } = props;
+
+  const generateToken = () => {
+    const token = getToken();
+    localStorage.token = token;
+    setToken(token);
+    if (token) navigate('/list');
+  };
+
+  return (
+    <div>
+      <h1>Welcome to your Smart Shopping List!</h1>
+      {token ? <Link to="/list">View your shopping list</Link> : null}
+      {!token ? (
+        <button onClick={() => generateToken()}>Create a new list</button>
+      ) : null}
+    </div>
```

Comment:
This looks great & I'm able to understand how to use the shopping-list-utils just by reading this so thank you! 

I'm wondering if the 2 ternary operators starting on line 18 could be combined into one?  I'm sure it's not a necessary fix but let me know your thoughts. 

Also, once this is merged, would it be fine to move this file into the pages folder we created? 

Great job overall!

Reply:
Good call, Viviana! I refactored that ternary, also renamed Home.js to Home.jsx and moved it to the pages folder so we're all on the same page structurally.

Reply:
Appreciate your work on the refactor, Rafa. Great catch, Viviana!